plugins/modules/proxysql_backend_servers.py

```diff
@@ -31,6 +31,10 @@
       - The port at which the mysqld instance can be contacted.
     type: int
     default: 3306
+  gtid_port:
+    description:
+      - The backend server port where ProxySQL Binlog Reader listens on for GTID tracking
+    type: int
```

Comment:
```suggestion
  gtid_port:
    description:
      - The backend server port where ProxySQL Binlog Reader listens on for GTID tracking
    type: int
    version_added: 1.5.0
```

It should also be mentioned in the description, that the default value is `0`, even if it's not set by the ansible module.

`    gtid_port INT CHECK ((gtid_port <> port OR gtid_port=0) AND gtid_port >= 0 AND gtid_port <= 65535) NOT NULL DEFAULT 0,`